docarray/base_doc/mixins/io.py

```diff
@@ -238,7 +242,7 @@ def from_protobuf(cls: Type[T], pb_msg: 'DocProto') -> T:
         for field_name in pb_msg.data:
             if (
                 not (cls.Config._load_extra_fields_from_protobuf)
-                and field_name not in cls.__fields__.keys()
+                and field_name not in cls._docarray_fields().keys()
```

Comment:
why a dictionary is accessed as a method?

Reply:
it is a method that return a dictionaries. I used initially a property but classmethd + property was failingwith python 3.8 (working with 3.9)